src/remote-config/remote-config-utils.ts

```diff
@@ -17,14 +17,14 @@
 import { PrefixedFirebaseError } from '../utils/error';
 
 export type RemoteConfigErrorCode =
-  'authentication-error'
-  | 'duplicate-key'
+  'aborted'
+  | 'already-exists'
+  | 'authentication-error'
```

Comment:
Not a OnePlatform error. What are we using this for? We should use `unauthenticated` or `permission-denied` instead.

Reply:
Good catch! Thanks. Fixed.